src/mpid/ch4/shm/src/topotree.c

```diff
@@ -11,14 +11,16 @@
 
 #include <math.h>
 
+#define MAX_TOPO_DEPTH 7
```

Comment:
Could this be the different levels of topo_depth?  https://github.com/pmodels/mpich/blob/main/src/include/mpir_hwtopo.h#L10 relates to topo_depth

Reply:
It depends. For example, L1 cache and L2 cache often is at the same level as core, and L3 and DDR is often at the same level of socket or CPU, etc.